src/Assets/TestProjects/KitchenSink/TestApp/TestApp.csproj

```diff
@@ -17,6 +17,9 @@
     <TieredCompilation>true</TieredCompilation>
     <TieredCompilationQuickJit>true</TieredCompilationQuickJit>
     <TieredCompilationQuickJitForLoops>true</TieredCompilationQuickJitForLoops>
+    <EventSourceSupport>false</EventSourceSupport>
+    <UseSystemResourceKeys>true</UseSystemResourceKeys>
```

Comment:
Some minor comment not directly from your PR is do we want to add some common prefix/sufix to these properties so that it is a bit obvious they are related and about the linker?

Reply:
> so that it is a bit obvious they are related and about the linker?

These options are not just about the linker - just like the `$(InvariantGlobalization)` property. These properties control runtime behavior. Setting these properties in any .NET 5 application will change its behavior as specified, since we set the value in the runtimeconfig.json. You don't need to run the linker in order to get the behavior. Yes, the linker recognizes and reacts to these properties, but to a user that is less important than the runtime behavior change.

Reply:
makes sense.

Reply:
One of the design goals of the feature switches is to define a general-purpose pattern we use for features - the linker's ability to drop code for disabled features is just one use case (you could in theory disable a feature without linking for example) - so I think we don't need a prefix.